### ISSUE TITLE
[FEATURE REQUEST] KVP or mysql

### ISSUE THREAD

**Issue by xrtw** [renzuzu/renzu_engine#12]
please make option if you want to save and load from KVP or MySQL

**Comment by renzuzu** [comment]
im still thinking if this is needed to convert SQL usage.

this only saves small string like, toysupmk4, elegy etc.. and plate .
if you want a plate changer this could be done easily , because if i convert this to SQL i will probably break the KVP data again like the stancer.

**Comment by xrtw** [comment]
> 

well that conversion was perfectly, no damage to my server.

**Comment by renzuzu** [comment]
right, ill put this on hold, i have a lot of backlogs in issues tickets. :D
and creating new resource too :D

ill try to do this week

**Comment by xrtw** [comment]
> right, ill put this on hold, i have a lot of backlogs in issues tickets. :D and creating new resource too :D
> 
> ill try to do this week

hey, possible to make something similar like renzu_stancer saving to mysql?
